cli/src/main.rs

```diff
@@ -4,8 +4,14 @@
 //! transactions within those blocks. This is only the CLI front-end. All the
 //! logic is contained in the library [`starknet_replay`].
 
-#![warn(clippy::all, clippy::cargo, clippy::pedantic)]
-#![allow(clippy::multiple_crate_versions)] // Due to duplicate dependencies in pathfinder
+#![warn(
+    clippy::all,
+    clippy::cargo,
+    clippy::pedantic,
+    clippy::missing_docs_in_private_items,
+    clippy::unwrap_used
+)]
+#![allow(clippy::multiple_crate_versions)] // Different libraries depend on different versions of the same library.
```

Comment:
This isn't a helpful comment. _Which_ libraries? Can we fix it?

Reply:
It's these many libraries:
```
multiple versions for dependency `base64`: 0.13.1, 0.21.7, 0.22.1
multiple versions for dependency `bitflags`: 1.3.2, 2.6.0
multiple versions for dependency `convert_case`: 0.4.0, 0.6.0
multiple versions for dependency `darling`: 0.14.4, 0.20.9
multiple versions for dependency `darling_core`: 0.14.4, 0.20.9
multiple versions for dependency `darling_macro`: 0.14.4, 0.20.9
multiple versions for dependency `hashbrown`: 0.12.3, 0.13.2, 0.14.5
multiple versions for dependency `indexmap`: 1.9.3, 2.2.6
multiple versions for dependency `itertools`: 0.10.5, 0.11.0
multiple versions for dependency `num-traits`: 0.1.43, 0.2.19
multiple versions for dependency `parking_lot`: 0.11.2, 0.12.3
multiple versions for dependency `parking_lot_core`: 0.8.6, 0.9.10
multiple versions for dependency `redox_syscall`: 0.2.16, 0.5.2
multiple versions for dependency `serde_with`: 2.3.3, 3.8.3
multiple versions for dependency `serde_with_macros`: 2.3.3, 3.8.3
multiple versions for dependency `starknet-core`: 0.5.1, 0.10.0
multiple versions for dependency `starknet-crypto`: 0.5.2, 0.6.2
multiple versions for dependency `starknet-curve`: 0.3.0, 0.4.2
multiple versions for dependency `strsim`: 0.10.0, 0.11.1
multiple versions for dependency `syn`: 1.0.109, 2.0.70
multiple versions for dependency `toml_edit`: 0.21.1, 0.22.15
multiple versions for dependency `windows-sys`: 0.48.0, 0.52.0
multiple versions for dependency `windows-targets`: 0.48.5, 0.52.6
multiple versions for dependency `windows_aarch64_gnullvm`: 0.48.5, 0.52.6
multiple versions for dependency `windows_aarch64_msvc`: 0.48.5, 0.52.6
multiple versions for dependency `regex-syntax`: 0.6.29, 0.8.4
multiple versions for dependency `windows_i686_gnu`: 0.48.5, 0.52.6
multiple versions for dependency `windows_i686_msvc`: 0.48.5, 0.52.6
multiple versions for dependency `windows_x86_64_gnu`: 0.48.5, 0.52.6
multiple versions for dependency `windows_x86_64_gnullvm`: 0.48.5, 0.52.6
multiple versions for dependency `windows_x86_64_msvc`: 0.48.5, 0.52.6
multiple versions for dependency `winnow`: 0.5.40, 0.6.13
multiple versions for dependency `base64`: 0.13.1, 0.21.7, 0.22.1
multiple versions for dependency `heck`: 0.3.3, 0.4.1, 0.5.0
multiple versions for dependency `regex-automata`: 0.1.10, 0.4.7
```